spec/diagnose_spec.rb

```diff
@@ -596,6 +675,35 @@
           "send_params" => true,
           "send_session_data" => true
         }
+      when :python
+        {
+          "app_path" => ending_with("appsignal-python"),
```

Comment:
This shouldn't really be in there (it's shouldn't be set by the user), but we can take it out later.

Reply:
Let's take it out later then